Canvas/GradesWidget/GradesTodayWidgetViewController.swift

```diff
@@ -128,24 +132,87 @@ class GradesTodayWidgetViewController: UIViewController {
 }
 
 extension GradesTodayWidgetViewController: UITableViewDataSource {
+    var SECTION_HEADER_HEIGHT: CGFloat {
+        return UIFont.scaledNamedFont(.bold20).lineHeight + VERTICAL_PADDING
+    }
+
+    var ASSIGNMENT_GRADE_ROW_HEIGHT: CGFloat {
+        return UIFont.scaledNamedFont(.medium12).lineHeight + UIFont.scaledNamedFont(.semibold16).lineHeight + VERTICAL_PADDING
+    }
+
+    var COURSE_GRADE_ROW_HEIGHT: CGFloat {
+        return UIFont.scaledNamedFont(.semibold16).lineHeight + VERTICAL_PADDING
+    }
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
```

Comment:
Do we have/need empty states for these sections? Does it look funny if the recently graded submissions are empty?

Reply:
Hmm we may want empty states. I will chat with dave.